### PR TITLE
[JAV-583] Pojo consumer return completable future

### PR DESCRIPTION
**Pull request by wujimin**

### CODE REVIEW

File: providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java [hunk 1]
```diff
@@ -86,20 +87,48 @@ protected void prepare() {
   @Override
   public Object invoke(Object proxy, Method method, Object[] args) throws Throwable {
     if (swaggerConsumer == null) {
-      prepare();
+      synchronized (this) {
+        if (swaggerConsumer == null) {
```
Comment (WillemJiang) [inline providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java]: swaggerConsumer should be volatile.

Comment (wujimin) [reply providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java]: done

Comment (WillemJiang) [reply providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java]: @wujimin  did you submit the change, I cannot find the change.

Comment (wujimin) [reply providers/provider-pojo/src/main/java/io/servicecomb/provider/pojo/Invoker.java]: maybe you did not refresh the page?

**Pull request merged**